src/databricks/labs/ucx/workspace_access/generic.py

```diff
@@ -299,9 +299,9 @@ def _crawl(self) -> list[WorkspaceObjectInfo]:
                 continue
             raw = obj.as_dict()
             yield WorkspaceObjectInfo(
-                object_type=raw["object_type"],
-                object_id=str(raw["object_id"]),
-                path=raw["path"],
+                object_type=raw.get("object_type", "Missing Object Type"),
```

Comment:
Can you rather do None instead of string with missing value? Also change the corresponding dataclass

Reply:
OK, Will do.